server/persistence/login.go

```diff
@@ -14,6 +15,22 @@ import (
 	"github.com/offen/offen/server/keys"
 )
 
+func (p *persistenceLayer) LoginSSO(email, salt string) (LoginResult, error) {
+	sha512Hash := sha512.New()
+	sha512Hash.Write([]byte(email))
+	sha512Hash.Write([]byte(salt))
+	dummyPassword := base64.URLEncoding.EncodeToString(sha512Hash.Sum(nil))
+	_, err := p.findAccountUser(email, false, false)
+	if err != nil {
```

Comment:
This should probably be more specific and only call `Join` when the error signals that no record was found. Joining should _not_ happen when for example network problem caused the query to fail.

Reply:
Fixed in the latest version